### PR TITLE
Adds performance indicators and exposes and endpoint for them

### PR DESCRIPTION
**Pull request by patrickberkeley**

### CODE REVIEW

**Comment by patrickberkeley** [comment]
//cc @alanpeabody or @jordpo

File: app/models/launch.rb [hunk 1]
```diff
@@ -1,3 +1,4 @@
 class Launch < Tile
-  validates_presence_of :start_date
+  validates :start_date,
+    presence: true
```
Comment (bmfay) [inline app/models/launch.rb:3]: curious about these above 4 changes.  just preference?

**Comment by bmfay** [comment]
LGTM, but i'd need some context on that rake test to say whether that LG or not.

File: app/models/launch.rb (hunk 1 above)
Comment (patrickberkeley) [inline app/models/launch.rb:3]: Yeah just a preference. Apparently this is the preferred way and I came across it while adding the validations for performance indicators.

**Comment by patrickberkeley** [comment]
@bmfay here's the google doc I'm ingesting for content areas, proficiencies, performance indicators: https://docs.google.com/spreadsheets/d/1iLmW_fgabEcXyiGMpUswJi6XR2289aJut86EuZ7n9IA/edit#gid=0

**Pull request merged**